### DISCUSSION TITLE
Feature Request: smoother transitions for background blur

### DISCUSSION THREAD

**Discussion by SynoUser-NL** [Clooos/Bubble-Card#425] [Feature requests]
Currently, when opening a PopUp, the background blur is activated immediately. The popup however is animated, sliding on to the screen. Smoothing the transition on the background blur, both in and out, for example during the slide in\out animation, would give this a much nicer feel.

**Comment by Clooos** [comment]
Hi! I will change that for the next release, good idea.

**Reply by SynoUser-NL** [reply]
Thanks very much!
